indico/htdocs/sass/modules/_contributions.scss

```diff
@@ -64,9 +64,9 @@
         display: flex;
 
         .label {
+            @include ellipsis;
```

Comment:
I wonder if the implicit `white-space: nowrap;` that comes with `@include ellipsis;` causes any problem. Did you test?